site/src/utils/useIsMobileView.ts

```diff
@@ -0,0 +1,10 @@
+import { useViewport } from "@jpmorganchase/uitk-core";
+
+const useIsMobileView = (): boolean => {
```

Comment:
should this be something more reusable? maybe useIsWithinViewport, idk, so you can pass the breakpoint you want to check for?